Sources/HeroContext.swift

```diff
@@ -139,6 +139,13 @@ extension HeroContext {
     unhide(view: view)
 
     // capture a snapshot without alpha, cornerRadius, or shadows
+    let oldMaskedCorners: CACornerMask = {
+      if #available(iOS 11, *) {
+        return view.layer.maskedCorners
+      } else {
+        return []
```

Comment:
Providing a default value here just to avoid working with an optional `CACornerMask`. Specifying all corners might be a better default here, but if this branch executes the value will never be applied to the layers anyway.